front/src/helpers/validateUrl.js

```diff
@@ -0,0 +1,13 @@
+const isUrl = (aUrl) => {
+  try {
+    new URL(aUrl);
+    return true;
+  } catch (_) {
+    return false;
+  }
+};
```

Comment:
estuve buscando otras opciones porque se ve rarito, pero como está encapsulado lo banco y las otras opciones no me terminaron de convencer (crear un anchor o usar una biblioteca `valid-url`). Lo investigué [acá](https://stackoverflow.com/questions/30931079/validating-a-url-in-node-js)

Reply:
O sea, se ve raro, pero quizas eliminado el catch de aca le puedan dar mejor forma